system-int-test/test/cmr/system_int_test/ingest/bulk_update/bulk_update_test.clj

```diff
@@ -29,6 +29,15 @@
                       :Topic "OCEANS"
                       :Term "MARINE SEDIMENTS"}]})
 
+(def science-keywords-umm-1
```

Comment:
my personal preference here is to make a more descriptive name, instead of adding a number to the end.
perhaps ```multiple-science-keywords-umm``` or something like.  Also, looking through the file, it seems there are already many defs for multiple science keywords, used in many different tests.  Maybe we should consolidate this in some way so we don't have an ever increasing list of refs used in random tests.

Reply:
I will change it to a more descriptive name. 
Consolidating will be more difficult because different tests have different needs. This particular one needs to contain a ScienceKeyword that exists in the update-value but not part of the keywords that's replaced.  
Personally I like to see different tests using different umm unless they need the exact same content, rather than keep adding more features to make a giant umm. It's easier to see what is expected.